gradle/version.properties

```diff
@@ -25,7 +25,7 @@ crate_admin_ui = 1.15.0
 crate_jdbc=2.5.1
 
 # ES deps
-elasticsearch=6.5.1
+elasticsearch=7.0.0-rc2
```

Comment:
We should make sure to get this to `7.0.0` before the feature branch will be merged.

Reply:
Yes will do now, was just released :-)